### PR TITLE
[MCKIN-8491] Engagement score average reflective of Discussion activity by Cohort Group

### PR DESCRIPTION
**Pull request by Agrendalath**
@UmanShahzad, this PR is a part of `[MCKIN-8491]` PR-s.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Agrendalath! I've created [OSPR-2686](https://openedx.atlassian.net/browse/OSPR-2686) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: edx_solutions_api_integration/courses/views.py [hunk 1]
```diff
@@ -1643,7 +1656,12 @@ def get(self, request, course_id):  # pylint: disable=W0613
...
-            users_enrolled_qs = CourseEnrollment.objects.users_enrolled_in(course_key).exclude(id__in=exclude_users)
+            try:
+                cohort_membership = CohortMembership.objects.get(user_id=user_id, course_id=course_key)
+                users_enrolled_qs = cohort_membership.course_user_group.users.exclude(id__in=exclude_users)
+            except ObjectDoesNotExist:
+                users_enrolled_qs = CourseEnrollment.objects.users_enrolled_in(course_key).exclude(
+                    id__in=exclude_users)
 
             if organization:
                 users_enrolled_qs = users_enrolled_qs.filter(organizations=organization).distinct()
```
[the remark this answers is not in the document]
Comment (Agrendalath) [reply edx_solutions_api_integration/courses/views.py]: Sorry, I forgot about these stats.

File: edx_solutions_api_integration/courses/views.py [hunk 2]
```diff
@@ -2052,11 +2095,19 @@ def get(self, request, course_id):
         course_key = get_course_key(course_id)
         # Users having certain roles (such as an Observer) are excluded from aggregations
         exclude_users = get_aggregate_exclusion_user_ids(course_key, roles=exclude_roles)
+        cohort_user_ids = None
+        try:
+            cohort_membership = CohortMembership.objects.get(user_id=user_id, course_id=course_key)
+            cohort_user_ids = cohort_membership.course_user_group.users.values_list('id', flat=True)
+        except ObjectDoesNotExist:
+            pass
```
[the remark this answers is not in the document]
Comment (Agrendalath) [reply edx_solutions_api_integration/courses/views.py]: @UmanShahzad, I've added the function in https://github.com/edx-solutions/edx-platform/pull/1191.

**Comment by UmanShahzad** [comment]
@Agrendalath Can you update setup.py here with version 2.0.10? I'll then merge and make a release tag.

**Comment by UmanShahzad** [comment]
@Agrendalath looks like this needs a rebase due to a conflict. And the latest version should be 2.0.10.

**Pull request merged**